src/manifest/manifest.ts

```diff
@@ -178,6 +178,11 @@ export default class Manifest extends EventEmitter<IManifestEvents> {
    */
   public isLive : boolean;
 
+  /**
+   * If true, the manifest will not be updated and the content is finished.
```

Comment:
I guess it could still technically be updated no?

Not in the currently-encountered case, but it just allows to say that the last possible chronological segment is available (at least that's how I see it).